pkg/sqlutil/monitor.go

```diff
@@ -130,8 +130,14 @@ func (q *queryLogger) logTiming(ctx context.Context, start time.Time) {
 	}
 	timeout := deadline.Sub(start)
 
-	pct := float64(elapsed) / float64(timeout)
-	pct *= 100
+	var pct float64
```

Comment:
/nit what does pct mean? 

`qtp` or `queryTimeoutPercentage` or even a comment would be a bit more descriptive for me

Reply:
Added a comment